xclim/indicators/atmos/_precip.py

```diff
@@ -132,7 +149,7 @@ class HrPrecip(Hourly):
     compute=indices.dry_days,
 )
 
-maximum_consecutive_wet_days = Precip(
+maximum_consecutive_wet_days = PrecipWithIndexing(
```

Comment:
I suppose this one as well as `maximum_consecutive_dry_days` are affected by the issue you described in https://github.com/Ouranosinc/xclim/issues/1069#issuecomment-1117540060 if indexer mask is applied before xclim starts counting the consecutive days.

Reply:
woups indeed!